### PR TITLE
Hotfix/752 watchdog notification

### PR DESCRIPTION
**Pull request by meox**

### CODE REVIEW

**Comment by meox** [comment]
I decided to use the HTTP Api because (for what I can see from code and documentation) is not possible to have a secondary Airbrake notifier.

This is important because we want to log Karma::Watchdog activities (:startup, :stop) on a dedicated Airbrake project and because this GEM is included as dependency inside PulsarPlatform we cannot override the default initializer.

**Comment by meox** [comment]
Already on staging
**Issue**: pulsarplatform/pulsarplatform#752

**Comment by meox** [comment]
Issue: https://github.com/pulsarplatform/pulsarplatform/issues/752

File: lib/karmap/watchdog.rb [hunk 1]
```diff
@@ -374,5 +380,36 @@ def sync_services_statuses
...
     end
+
+    #
+    # Send a notification to AirBrake (using HTTP Api)
+    # use ENV['AIRBRAKE_KARMA_PROJECT_ID'] and ENV['AIRBRAKE_KARMA_PROJECT_KEY']
+    #
+    # @param [String] notification: message of the notification
+    # @return [Bool] true if notification is sent (with success), false otherwise
+    #
+    def send_airbrake_notification(notification_type = 'notification')
```
Comment (abrazzini) [inline lib/karmap/watchdog.rb:391]: sync parameters with documentation
the method seems to require notification param
check also the call (raw: 118)

File: lib/karmap/watchdog.rb [hunk 2]
```diff
@@ -110,6 +115,7 @@ def run
         sleep ONE_SECOND
       end
 
+      send_airbrake_notification('stop')
```
Comment (abrazzini) [inline lib/karmap/watchdog.rb:118]: check the params

File: lib/karmap/watchdog.rb [hunk 3]
```diff
@@ -99,6 +102,8 @@ def run
       start_queue_poller
       init_traps
 
+      send_airbrake_notification('startup')
```
Comment (abrazzini) [inline lib/karmap/watchdog.rb:105]: check the param

**Review by abrazzini — CHANGES_REQUESTED** [review]

**Pull request merged**